sklearn/utils/__init__.py

```diff
@@ -1090,3 +1094,129 @@ def check_pandas_support(caller_name):
         raise ImportError(
             "{} requires pandas.".format(caller_name)
         ) from e
+
+
+def all_estimators(include_meta_estimators=None,
```

Comment:
why do we need this here instead of importing from `utils._testing`?

Reply:
Circular imports :'( 

base uses utils
utils.testing uses lots of things from base

These imports are now lazy in `all_estimators`

Reply:
got it.